src/applications/claims-status/components/claim-overview-tab/ClaimPhaseStepper.jsx

```diff
@@ -70,12 +70,7 @@ export default function ClaimPhaseStepper({
             evidence after this step, your claim will go back to this step for
             review.
           </p>
-          <Link
-            data-testid="submit-evidence-link"
-            aria-label="Submit evidence now"
-            title="Submit evidence now"
-            to="../files"
-          >
+          <Link data-testid="submit-evidence-link" to="../files">
```

Comment:
Updated active link to no longer have a title or aria-label since its not needed